cmd/genutils/genutils.go

```diff
@@ -41,3 +43,17 @@ func OutDir(path string) (string, error) {
 	outDir = outDir + "/"
 	return outDir, nil
 }
+
+// ParseKubeConfigFiles gets a string that contains one or multiple kubeconfig files.
+// If there are more than one file, separated by comma
+// Returns an array of filenames whose existence are validated
+func ParseKubeConfigFiles(kubeConfigFilenames string) ([]string, bool) {
+	kubeConfigs := strings.Split(kubeConfigFilenames, ",")
+	for _, kubeConfig := range kubeConfigs {
+		_, err := os.Stat(kubeConfig)
+		if err != nil {
```

Comment:
suggest to return error instead of bool

Reply:
Error can have different type. Hence caller needs to differentiate error type and perform different actions. Since there are many usages for this function, keep it as return bool for simplified follow up actions.